Aesop/BuiltinRules/DestructProducts.lean

```diff
@@ -80,7 +80,7 @@ partial def destructProductsCore (goal : MVarId) : MetaM MVarId :=
         else
           return goal
 
-elab &"aesop_destruct_products" : tactic =>
+elab "aesop_destruct_products" : tactic =>
```

Comment:
It's not necessary to put `&` in front of the initial atom in a category with `leading_ident_behavior := symbol` or `leading_ident_behavior := both`, and `tactic` and `attr` are both of that form.